src/user_defines.txt

```diff
@@ -71,3 +71,7 @@
 # Uncomment to use the inverted sbus pad on the r9mm receiver and flight controller.  'set serialrx_inverted = ON' may also need to be set within Betaflight.
 # TLM is not available.  Flashing via the BetaflightPassthrough target will not work!!!
 #-DUSE_R9MM_R9MINI_SBUS
+
+# Any OpenTX channel (defaults to ch16) can be used for setting the TX power level by uncommenting the option below.
+# This allows you to change the TX power level during flight using a switch / pot in OpenTX
+#-DUSE_DYNAMIC_POWER
```

Comment:
To make it easy for users to select the channel, we can remove this USE_DYNAMIC_POWER and only use DYNAMIC_POWER_CHANNEL=15.